src/main/java/org/osiam/storage/query/GroupSimpleFilterChain.java

```diff
@@ -33,25 +31,25 @@
 
 public class GroupSimpleFilterChain implements FilterChain<GroupEntity> {
 
-    private final ScimExpression scimExpression;
+    private final FilterExpression filterExpression;
 
     private final GroupQueryField filterField;
     private final CriteriaBuilder criteriaBuilder;
 
-    public GroupSimpleFilterChain(CriteriaBuilder criteriaBuilder, ScimExpression scimExpression) {
+    public GroupSimpleFilterChain(CriteriaBuilder criteriaBuilder, FilterExpression filterExpression) {
         this.criteriaBuilder = criteriaBuilder;
-        this.scimExpression = scimExpression;
-        filterField = GroupQueryField.fromString(scimExpression.getField().toLowerCase(Locale.ENGLISH));
+        this.filterExpression = filterExpression;
+        filterField = GroupQueryField.fromString(filterExpression.getField().getFieldName());
     }
 
     @Override
     public Predicate createPredicateAndJoin(Root<GroupEntity> root) {
         if (filterField == null) {
```

Comment:
You can remove this guard clause, if you follow the [previous comment](https://github.com/osiam/osiam/pull/108/files#r49906584).